doc/doxygen/src/Examples.dox

```diff
@@ -277,9 +277,14 @@
  *
  * # Whenever we make new data, we always tell the manager first,
  * # This allows it to create a placeholder version or similar.
+ * # We must provide the manager with as much information as possible
+ * # up-front, to help it prepare the backend and determine the most
+ * # appropriate response to our subsequent `resolve` and `register`
+ * # calls, or error early if publishing is likely to fail.
```

Comment:
[Question] "As much information as possible up front" ... I wonder if this is correct. Do we want to say something more like, "We must provide the manager with _all_ the information known about the entity"? I know they technically say the same thing, but the latter implies less wiggle room.

Reply:
[fix] I think the key wording we worked out before was about property values that the host _owns_ vs ones it will try to resolve from the manager as part of publish (more on this in the issue).

Reply:
Shortened and used the word "owns" in daa8a1ac